{"name": "test_login_params[data_for_login2]", "status": "passed", "attachments": [{"name": "log", "source": "e28a5458-865d-4eda-9395-f3da0fde0429-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('adelukesh@gmail.com', '48465', 'Fail')"}], "start": 1745571064183, "stop": 1745571067139, "uuid": "bc2f2c6a-9fea-4354-8cef-1f5e777492c3", "historyId": "cd1ac65cf3cc05caccdeb520d07aec8a", "testCaseId": "47f970e602e4eefb1a3e5455dd555a36", "fullName": "testCases.test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "26028-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login_params"}]}